src/NuGetPackageConfigConverter/Services/PackageManagerConverter.cs

```diff
@@ -272,4 +392,53 @@ private static ProjectItem GetProjectItem(ProjectItems items, string name)
             return null;
         }
     }
+
+
+
+    static class ProjectExtensions
+    {
+        public static string GetFullName(this Project project)
```

Comment:
This method already exists in `VSExtensions`; it just needs to be made public and an extension method